db/seeds.rb

```diff
@@ -11,12 +11,12 @@
 u.email = "wts@wts.com"
 u.password='swordfish'
 u.password_confirmation='swordfish'
+u.role = 'SuperAdmin',
 u.save
 
 # Create the user's game
 u.game= Game.create(
   name: "Watch the Skies Test",
-  role: "SuperAdmin",
```

Comment:
This was causing problems for me when I tried to reset to seed data, because the role is on the user account, not on the game model.